### PR TITLE
docs: Update Japanese translation 20190822

### PR DESCRIPTION
**Pull request by ibu1224**
## Context

Update Japanese translation. 
```
docs/ja/about/appendix/domain.md
docs/ja/user-guide/configuration/externalConfig.md
```
## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by kumada626** [comment]
You should delete `並列実行` section in domain.md. That section is removed from English docs.
https://github.com/screwdriver-cd/guide/pull/317/files

**Comment by kumada626** [comment]
It is better to translate workflow.md too.
https://github.com/screwdriver-cd/guide/pull/318/files

File: docs/ja/about/appendix/domain.md [hunk 1]
```diff
@@ -125,25 +125,29 @@ $ meta get example
 
 ### Workflow
 
-ワークフローとは、`main`ジョブの[ビルド](#%E3%83%93%E3%83%AB%E3%83%89)成功後に実行される[ジョブ](#%E3%82%B8%E3%83%A7%E3%83%96)の順番のことです。`main`ジョブは常に最初に実行されます。ジョブは並列や逐次、またはその組み合わせで実行することができます。ワークフローにはパイプライン内に定義されたジョブがすべて含まれている必要があります。
+[ワークフロー](../../user-guide/convfiguration/workflow)とは、初めのジョブの[ビルド](#%E3%83%93%E3%83%AB%E3%83%89)成功後に実行される[ジョブ](#%E3%82%B8%E3%83%A7%E3%83%96)の順番のことです。ジョブは並列や逐次、またはその組み合わせで実行することができます。順序は`requires`キーで決定されます。
```
Comment (yoshwata) [inline docs/ja/about/appendix/domain.md]: convfiguration

Comment (ibu1224) [reply docs/ja/about/appendix/domain.md]: fixed!

File: docs/ja/user-guide/configuration/workflow.md [hunk 2]
```diff
@@ -294,6 +294,7 @@ jobs:
 
 以下の例では、`detached-main` ジョブが分離されています。
 
+
```
Comment (kumada626) [inline docs/ja/user-guide/configuration/workflow.md]: An useless line.

Comment (ibu1224) [reply docs/ja/user-guide/configuration/workflow.md]: oh.. fixed

**Review by yoshwata — APPROVED** [review]
LGTM

**Review by kumada626 — APPROVED** [review]
👍

**Review by tkyi — APPROVED** [review]

**Pull request merged**